master/static/migrations/20231208132223_drop-raw-checkpoints.tx.up.sql

```diff
@@ -0,0 +1,24 @@
+ALTER TABLE checkpoints_v2 
+    ALTER COLUMN id DROP DEFAULT;
+
+-- We need to add `IF EXISTS` to these data drops in order to safely perform down migrations. 
+-- Since this migration does not have a corresponding down migration, we need to verify existence
+-- of the following functions, table, and sequence in order to migrate up and down from this 
+-- schema version.
+
+DROP FUNCTION IF EXISTS best_checkpoint_by_metric;
+DROP FUNCTION IF EXISTS experiments_best_checkpoints_by_metric;
+DROP TABLE IF EXISTS raw_checkpoints;
+DROP SEQUENCE IF EXISTS checkpoints_id_seq;
+
+CREATE SEQUENCE checkpoints_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+SELECT setval('checkpoints_id_seq', (SELECT MAX(id) FROM checkpoints_v2));
```

Comment:
Do we need `coalesce((SELECT MAX(id) FROM checkpoints_v2), 1)`, in case MAX(id) returns NULL?

Reply:
Hmm, I initially had this, but it causes `nextval` to start from 2 when `setval` starts from 1. Changing the `COALESCE` default value to start from 0 also causes problems because `MINVALUE` for a sequence is 1.
I did check manually, and if our select returns `NULL`, inserting into `checkpoints_v2` will still create a row with a default id starting at 1.

Reply:
Actually, I'm testing this manually and it seems like even without `COALESCE`, insertion starts the id sequence at 2 sometimes. I'll add back `COALESCE` and will  look more into this! Good catch Jerry!

Reply:
Okay, last comment sorry lol false alarm. I forgot to wipe my db after testing with `COALESCE`, but removing it seems to keep id sequence correctly starting at 1 even with a `NULL` return value from the `MAX(id)` select statement. 
So, will probably leave it as is.